roles/rsyslog/vars/RedHat.yml

```diff
@@ -0,0 +1 @@
+__rsyslog_input_basics: input_basics.j2
```

Comment:
this should be in `roles/rsyslog/vars/default.yml`

Reply:
Ok, I'm going to add default.yml. Had we better remove RedHat.yml (and its symlink CentOS.yml) for now?

Reply:
> Ok, I'm going to add default.yml. Had we better remove RedHat.yml (and its symlink CentOS.yml) for now?

Yes, please

Reply:
Thank you, @richm. Pushed the fix.